src/templates/back/@api/__module_name__/controllers/__bounded_context_name__-find-__module_name__-by-id.controller.ts

```diff
@@ -1,22 +1,38 @@
-/* eslint-disable @typescript-eslint/explicit-function-return-type */
-import { Body, Controller, HttpCode, Param, Post } from '@nestjs/common';
-import { ApiTags, ApiOkResponse, ApiOperation } from '@nestjs/swagger';
-import { {{#if schema.properties.hasI18n}}ContentLanguage, {{/if}}QueryStatement, Timezone } from '{{ config.auroraCorePackage }}';
-import { {{ toPascalCase schema.boundedContextName }}{{ toPascalCase schema.moduleName }}Dto } from '../dto';
+{{
+    setVar 'importsArray' (
+        array
+            (object items=(array 'Body' 'Controller' 'HttpCode' 'Param' 'Post')  path='@nestjs/common')
+            (object items=(array 'ApiTags' 'ApiOkResponse' 'ApiOperation')  path='@nestjs/swagger')
```

Comment:
Mantener un orden alfabético
```suggestion
            (object items=(array 'ApiOkResponse' 'ApiOperation' 'ApiTags')  path='@nestjs/swagger')
```